.github/workflows/Manager-CI.yaml

```diff
@@ -103,6 +97,28 @@ jobs:
           sed -i -E "s|(.*MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN=).*|\1$MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN|" kustomization.yaml
           sed -e ':a;s/^\(  - [A-Z_]\+_SECRET\=[^=]\{4\}x*\)[^x]/\1x/;t a' -e ':b;s/^\(  - [A-Z_]\+\=[^=]\{12\}x*\)[^x]/\1x/;t b' kustomization.yaml
         working-directory: kustomize/overlays/minikube/manager
+      - name: Configure Red Hat SSO for shard
+        shell: bash
+        env:
+          DEV_EVENT_BRIDGE_SSO_CLIENT_ID: ${{ secrets.DEV_EVENT_BRIDGE_SSO_CLIENT_ID }}
+          DEV_EVENT_BRIDGE_SSO_SECRET: ${{ secrets.DEV_EVENT_BRIDGE_SSO_SECRET }}
+          DEV_WEBHOOK_TECHNICAL_ACCOUNT_ID: ${{ secrets.DEV_WEBHOOK_TECHNICAL_ACCOUNT_ID }}
+          DEV_WEBHOOK_CLIENT_ID: ${{ secrets.DEV_WEBHOOK_CLIENT_ID }}
+          DEV_WEBHOOK_CLIENT_SECRET: ${{ secrets.DEV_WEBHOOK_CLIENT_SECRET }}
+        run: |
+          sed -i -E "s|(.*EVENT_BRIDGE_SSO_CLIENT_ID=).*|\1$DEV_EVENT_BRIDGE_SSO_CLIENT_ID|" kustomization.yaml
```

Comment:
I guess it may be possible, just need to use same key for SealedSecret, that would have to be stored in GH Secret.

Reply:
@sutaakar Yeah that was my thought. Basically I think it would be good for us to align across all our environments as closely as possible so that things are common for the team.

Reply:
I would try to move everything to Vault so that we have all configured in the same way (also downstream).
Btw we can probably do also SealedSecret but we need to [extract the private key](https://github.com/bitnami-labs/sealed-secrets#how-can-i-do-a-backup-of-my-sealedsecrets) (in a GH Secret?) and applied while we create the cluster in the pipeline (the `kind` cluster is created everytime so we need to restore the private key to decrypt the secret)